awx/settings/defaults.py

```diff
@@ -573,9 +573,6 @@ def IS_TESTING(argv=None):
 # Additional environment variables to be passed to the ansible subprocesses
 AWX_TASK_ENV = {}
 
-# Flag to enable/disable updating hosts M2M when saving job events.
-CAPTURE_JOB_EVENT_HOSTS = False
```

Comment:
This was the original shortcut to *not* do this if someone needed to eek out performance gains.

Reply:
Looks like it got turned off years ago 🤷‍♂

Reply:
https://github.com/ansible/awx/blame/devel/awx/settings/defaults.py#L577